src/platform/symbols.map

```diff
@@ -1,4 +1,4 @@
-MIR_PLATFORM_2.18 {
+MIR_PLATFORM_2.19 {
  global:
```

Comment:
I don't think we're breaking the mirplatform ABI: `idle_timeout_on_lock_opt` should be in a new 2.19 stanza